src/main/java/me/shivzee/util/Utility.java

```diff
@@ -24,5 +28,22 @@ public static String createRandomString(int length){
         return randomString;
     }
 
+    public static ZonedDateTime parseDateTimeToDefaultTimeZone(String dateTime, String pattern) {
+        try {
+            dateTime = dateTime.replace("am", "AM");
+            dateTime = dateTime.replace("pm", "PM");
+            LocalDateTime objLocalDateTime = LocalDateTime.parse(dateTime, DateTimeFormatter.ofPattern(pattern));
+            return objLocalDateTime.atZone(ZoneId.systemDefault());
+        }catch(Exception ex) {
+            try {
+                dateTime = dateTime.replace("AM", "am");
+                dateTime = dateTime.replace("PM", "pm");
+                LocalDateTime objLocalDateTime = LocalDateTime.parse(dateTime, DateTimeFormatter.ofPattern(pattern));
+                return objLocalDateTime.atZone(ZoneId.systemDefault());
+            }catch(Throwable t) {
+            }
+        }
+        return null;
+    }

```

Comment:
badly written function 
the `parseDateTimeToDefaultTimeZone` can be written in short as `parseToDefaultTimeZone` 
and the function can be simplified as
```java
public static ZonedDateTime parseToDefaultTimeZone(String dateTime, String pattern) {
        ZonedDateTime time = null;
        try {
            time = LocalDateTime.parse(dateTime, DateTimeFormatter
            .ofPattern(pattern))
            .atZone(ZoneId.systemDefault());
        }catch(Exception ex) {
            //TODO -> Add the LOG.Error();
        }
        return time;
    }
```
there is no need for `AM`,`PM` its a UTC timestamp which mail.tm api follows
also add a little bit info in the documentation comment above for better understanding of what this function does

```suggestion
        ZonedDateTime time = null;
        try {
            time = LocalDateTime.parse(dateTime, DateTimeFormatter
            .ofPattern(pattern))
            .atZone(ZoneId.systemDefault());
        }catch(Exception ex) {
            //TODO -> Add the LOG.Error();
        }
        return time;
        }

```